docs/source/faq.rst

```diff
@@ -0,0 +1,80 @@
+Issues & FAQ
+===================================
+
+
+FAQ
+########
+
+Issues 
+########
+
+
+Java JVM DLL not found (specific to Mac)
+**********************
+
+One user reported the following error when installing `pyspi` on a MacBook Air an M2 chip and Catalina OS:
+
+.. code-block::
+
+   OSError: [Errno 0] JVM DLL not found /Library/Java/JavaVirtualMachines/jdk-19.jdk/Contents/Home/lib/libjli.dylib
+
+
+This issue is similar to those reported `here <https://stackoverflow.com/questions/71504214/jvm-dll-not-found-but-i-can-clearly-see-the-file>`_ and `here <https://github.com/jpype-project/jpype/issues/994>`_. This issue can arise from the version of OpenJDK identified as the system default
```

Comment:
Missed a fullstop. Text might also flow a little better by using a semi-colon? Something like 

This issue is similar to those reported `here <https://stackoverflow.com/questions/71504214/jvm-dll-not-found-but-i-can-clearly-see-the-file>`_ and `here <https://github.com/jpype-project/jpype/issues/994>`_; it can arise from the version of OpenJDK identified as the system default.

Reply:
Thanks for pointing that out, amended with your suggestion :)